src/pymor/operators/numpy.py

```diff
@@ -15,14 +15,15 @@
 
 from functools import reduce
 
+from math import ceil
```

Comment:
I think we usually arrange imports by three sections: standard library, third party, first party.

```suggestion
from functools import reduce
from math import ceil

```